types/vis/vis-tests.ts

```diff
@@ -34,7 +37,7 @@ data.add([
 ]);
 
 // subscribe to any change in the DataSet
-data.on('*', (event, properties, senderId) => {
+data.on('*', (event: any, properties: any, senderId: any) => {
```

Comment:
do not use any

Reply:
This in fact is not related with this pull request, but hooks (which have rules newer than latest vis change) are failing because implicit type warning. This is just to make hooks happy, I really dont think this should be a blocker for this PR because in fact i dont know what should be there.

Reply:
Revert the changes then.

Reply:
The thing is that then hooks will fail, and it won't be able to merge even this.

Hooks fails because you shouldn't use implicit detection. This solves it just enough to pass the hooks.

Reply:
We can open a separate ticket (even without this PR coming in) that types should be handled here. Because from now on, every PR hook will fail because previously wasn't done properly, and in the meanwhile definitely typed introduced different hooks